build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext.versions = [
-            'compileSdk'         : 27,
+            'compileSdk'         : 28,
             'minSdk'             : 15,
-            'targetSdk'          : 27,
-            'androidGradlePlugin': '3.1.3',
+            'targetSdk'          : 28,
```

Comment:
❓Were you able to test that none of the functionality is broken after targeting SDK 28?

Reply:
Yes, tested on api level 15, 21, 27 and 28. All functionality still works